src/parser/FStar.Parser.Dep.fs

```diff
@@ -1397,6 +1397,15 @@ let print_raw (deps:deps) =
       |> String.concat ";;\n"
       |> BU.print_endline
 
+(* Do I need to print dependencies for .ml, .cmx, etc.? *)
+
+let dep_opt_le (lesser: string) (greater: string) : bool =
```

Comment:
this could use either a comment describing the order relation, or a match-over-pairs pattern that makes it easier to comprehend